docs/changes/coordinates/17263.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Fix a crash when instantiating ``Angle`` (or ``Latitude``, or ``Longitude``)
```

Comment:
Did you verify that the bug was present with `Latitude` and `Longitude` and that it's fixed now? I didn't bother installing `pyarrow` for this, so I haven't verified it myself.

Reply:
I did.